doc/schemas/gov_openapi.json

```diff
@@ -1605,6 +1626,28 @@
         }
       }
     },
+    "/gov/kv/interpreter/flush": {
```

Comment:
Does this flush the interpreter states? If so, `get` doesn't seem like the most obvious choice.

Reply:
No - this is an auto-generated endpoint returning the current value from the governance table.